integration/setup.go

```diff
@@ -366,7 +385,7 @@ func (ts *testSetup) waitUntilServerIsDown() error {
 func (ts *testSetup) startServer() error {
 	log := ts.storageOpts.InstrumentOptions().Logger()
 	fields := []xlog.Field{
-		xlog.NewField("nativepooling", ts.nativePooling),
+		xlog.NewField("cachepolicy", ts.storageOpts.SeriesCachePolicy().String()),
```

Comment:
Considering we do not run native pooling in any tests/anywhere by default anymore I don't see the reason to - it also takes up significant space in the logs since this is attached to every log line.

Reply:
fair enough